pkgs/top-level/all-packages.nix

```diff
@@ -971,7 +971,8 @@ with pkgs;
   makeShellWrapper = makeSetupHook
     { deps = [ dieHook ];
       substitutions = {
-        shell = targetPackages.runtimeShell;
+        # targetPackages.runtimeShell only exists when pkgs == targetPackages (when targetPackages is not  __raw)
```

Comment:
What does targetPackages is not __raw mean?

Reply:
https://github.com/NixOS/nixpkgs/blob/2622e90437c7e30c08a8edd80ffbcdd78edc7493/pkgs/stdenv/booter.nix#L118-L140 the set only contains stdenv